perf/specjbb/build/getSpecjbbSuite.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+
+SUCCESS=0
+ERROR=1
+
+exists() {
+    local specjbb_dir=$1
+    return $(test -f "${specjbb_dir}/${SPECJBB_JAR}")
+}
+
+isRunnable() {
+    local specjbb_dir=$1
+    #result=$(${JAVA_BIN} -jar ${specjbb_dir}/${SPECJBB_JAR} -v > /dev/null 2>&1 | grep -q "SPECjbb2015") > /dev/null 2>&1
+    result=$(${JAVA_BIN} -jar ${specjbb_dir}/${SPECJBB_JAR} -v | grep -q "SPECjbb2015") > /dev/null 2>&1
+    return $result
+}
+
+dirIsValid(){
+    local specjbb_dir=$1
+
+    if exists $specjbb_dir; then
+        if isRunnable $specjbb_dir; then
+            return $SUCCESS
+        fi
+    fi
+
+    return $ERROR
+}
+
+dstIsValid(){
+    echo "##### Validating destination suite"
```

Comment:
Is "suite" the right word here and below? Are these functions validating the directories?